### ISSUE TITLE
Update lizmap-documentation from the latest transifex for Japanese language

### ISSUE THREAD

**Issue by JinIgarashi** [3liz/lizmap-documentation#180]
We completed to translate all of documentation in Japanese language at transifex.

Could you update this documentation site (https://docs.lizmap.com/next/ja/index.html) from the latest translation?

**Comment by Gustry** [comment]
I just did it.
Thanks a lot for your contributions !
Let us know what you are doing with Lizmap if you can ;-)

**Comment by JinIgarashi** [comment]
@Gustry Thank you, however the translation does not seem reflected.
There are still a lot of English text remained...Could you confirm it?

https://docs.lizmap.com/next/ja/

https://docs.lizmap.com/next/ja/publish/index.html

https://docs.lizmap.com/next/ja/install/index.html

**Comment by Gustry** [comment]
I have raised the issue to our team. We will keep you up to date.

**Comment by JinIgarashi** [comment]
@Gustry how about this issue? I want to check our translations in website.

**Comment by JinIgarashi** [comment]
@Gustry How about the progress of this issue? Is it so serious? I just want to confirm our translations in the production site. Or if you have any testing site and translations are already reflected, could you give the link?

**Comment by Gustry** [comment]
Can you try again now ? It seems better now.
I have updated master, lizmap 3.3 and 3.4.

But maybe you didn't translate well when there `:guilabels:` ?
https://docs.lizmap.com/next/ja/publish/lizmap_plugin/layers.html 

The string
```
:guilabel:`Image format`:
```
must have been translated into
```
:guilabel:`画像形式`:‎
```
And not in
```
:guilabel:'画像形式':‎
```

**Comment by JinIgarashi** [comment]
@Gustry Thank you. It's better now. About the issue of `:guilabel`, I am fixing them now.

**Comment by Gustry** [comment]
Hum, maybe, we should do it with CLI tools ?
 Because there is a lot to fix.

**Comment by Gustry** [comment]
By the way, it seems some of Japanese people tried to translate Javanese and not Japanese. I think this was a mistake ;-)
I have removed the Javanese language from Transifex, and maybe you want to invite your colleagues to double check ;-)

**Comment by JinIgarashi** [comment]
These translations were done by another translator. I think it is not easy to use CLI tools, some mistakes are complicated. Let me done manually.

I didn't know that some Japanese translated Javanese...yeah you can delete that.

**Comment by JinIgarashi** [comment]
@Gustry I fixed master and 3.4 for the issue related to `:guilabel`. He translated strings under `publishpot`. Could you simply regenerate `lizmap-3-3--publishpot` which you have added today? so at least my fixed `:guillabel` can be better in 3.3.

**Comment by Gustry** [comment]
Sorry, lizmap 3.3 was a push mistake. We won't release any 3.3 release.

You should focus on `master` and `3.4`.
Master will be Lizmap 3.5 soon. We are going to release LWC 3.5.0 by the end of september.

I have just pushed again :

* https://docs.lizmap.com/current/ja/ equivalent to https://docs.lizmap.com/ : current lizmap 3.4
* https://docs.lizmap.com/current/ja/ : next Lizmap 3.5

**Comment by JinIgarashi** [comment]
Thank you. I see your releasing schedule. 
I also found some translation errors in my part. I will try to fix them as soon as I can.

**Comment by Gustry** [comment]
> Thank you. I see your releasing schedule.

We have released RC 2 yesterday for 3.5.
We still have one feature (and some strings) to merge.

I'm closing as it's seems fixed. Thanks for your contribution.
